packages/replay/src/coreHandlers/handleHistory.ts

```diff
@@ -1,5 +1,5 @@
-import { ReplayPerformanceEntry } from '../createPerformanceEntry';
 import type { ReplayContainer } from '../types';
+import { ReplayPerformanceEntry } from '../util/createPerformanceEntry';
```

Comment:
Is there no lint rule to always `import type` for types?

Reply:
No, but it's a good idea - I'll look into it!

Reply:
See: https://github.com/getsentry/sentry-javascript/pull/6662